apps/gp2-frontend/src/__tests__/OnboardedApp.test.tsx

```diff
@@ -40,7 +40,10 @@ const renderAuthenticatedApp = async () => {
         <Auth0Provider user={{ id }}>
           <WhenReady>
             <StaticRouter>
-              <OnboardedApp />
+              <OnboardedApp
+                showWelcomeBackBanner={false}
+                dismissBanner={jest.fn()}
```

Comment:
I can't see any tests displaying the banner and hiding it?

Reply:
on the test we mock the dashboard page, so we don't have a banner to click in